src/main/java/com/google/codeu/data/Message.java

```diff
@@ -47,12 +48,17 @@ public Message(UUID id, String user, String text, long timestamp, String recipie
     this.text = text;
     this.timestamp = timestamp;
     this.recipient = recipient;
+    this.imageUrl = null;
```

Comment:
if think if you don't set this.imageUrl, it defaults to null, so i think this line may be redundant -- I would check to make sure though!

Reply:
Just checked and yep, it's set to null. My C background always seems to make an appearance.